recipes/sdbus-cpp/all/conanfile.py

```diff
@@ -0,0 +1,109 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SdbusCppConan(ConanFile):
+    name = "sdbus-cpp"
+    license = "LGPL-2.1-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Kistler-Group/sdbus-cpp"
+    description = "High-level C++ D-Bus library for Linux designed" \
+                  " to provide easy-to-use yet powerful API in modern C++"
+    topics = ("conan", "dbus", "sd-bus", "sdbus-c++", "sdbus-cpp")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_code_gen": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_code_gen": False,
+    }
+    generators = "pkg_config"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Only Linux supported")
+        if self.options.shared:
+            del self.options.fPIC
+        tools.check_min_cppstd(self, "17")
+
+    def build_requirements(self):
+        self.build_requires("cmake/3.19.1")
```

Comment:
We usually don't require CMake as this is one of very few dependencies we pre-require. Exceptions can be made when the CMake version required is very recent or only supports a specific old versions or something like that.

This projects seems to be fine with CMake >= 3.13.0 so I would argue that it should be removed here

https://github.com/Kistler-Group/sdbus-cpp/blob/d6fdacafbe5e490d540956fe78f8df808723f49e/CMakeLists.txt#L5